### ISSUE TITLE
1.18.2

### ISSUE THREAD

**Issue by UsainSrht** [ShaneBeee/SkBee#141]
SkBee doesn't work on 1.18.2
```
[23:38:45] [Server thread/ERROR]: Error occurred while enabling SkBee v1.15.0 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
	at com.shanebeestudios.skbee.api.NBT.NBTCompound.mergeCompound(NBTCompound.java:86) ~[SkBee-1.15.0.jar:?]
	at com.shanebeestudios.skbee.api.NBT.NBTApi.forceLoadNBT(NBTApi.java:104) ~[SkBee-1.15.0.jar:?]
	at com.shanebeestudios.skbee.api.NBT.NBTApi.<init>(NBTApi.java:49) ~[SkBee-1.15.0.jar:?]
	at com.shanebeestudios.skbee.SkBee.onEnable(SkBee.java:97) ~[SkBee-1.15.0.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:501) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugin(CraftServer.java:559) ~[purpur-1.18.2.jar:git-Purpur-1574]
	at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugins(CraftServer.java:473) ~[purpur-1.18.2.jar:git-Purpur-1574]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:671) ~[purpur-1.18.2.jar:git-Purpur-1574]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:438) ~[purpur-1.18.2.jar:git-Purpur-1574]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:352) ~[purpur-1.18.2.jar:git-Purpur-1574]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1172) ~[purpur-1.18.2.jar:git-Purpur-1574]
	at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:322) ~[purpur-1.18.2.jar:git-Purpur-1574]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Class.getDeclaredField(String)" because the return value of "com.shanebeestudios.skbee.api.NBT.utils.nmsmappings.ClassWrapper.getClazz()" is null
	at com.shanebeestudios.skbee.api.NBT.NBTReflectionUtil.<clinit>(NBTReflectionUtil.java:35) ~[SkBee-1.15.0.jar:?]
	... 15 more
[23:38:45] [Server thread/INFO]: [SkBee] Disabling SkBee v1.15.0
```

**Comment by Lotzyprod** [comment]
Guys you can change dependency version of **item-nbt-api-plugin** to **2.9.2** in **pom.xml**
```
<groupId>de.tr7zw</groupId>
<artifactId>item-nbt-api</artifactId>
<version>2.9.2-SNAPSHOT</version>
```
and compile it with maven by himself with command **mvn clean install** in main directory of Skbee project

**Comment by AlphaKR93** [comment]
> Guys you can change dependency version of **item-nbt-api-plugin** to **2.9.2** in **pom.xml**
> 
> ```
> <groupId>de.tr7zw</groupId>
> <artifactId>item-nbt-api</artifactId>
> <version>2.9.2-SNAPSHOT</version>
> ```
> 
> and compile it with maven by himself with command **mvn clean install** in main directory of **Skbee** project
> 
> attach compiled version for lazy users :) [SkBee-1.15.0-1.18.2.zip](https://github.com/ShaneBeee/SkBee/files/8192715/SkBee-1.15.0-1.18.2.zip)

Thanks!!!

**Comment by ShaneBeee** [comment]
fixed in latest release, 1.15.1
